stdlib/stdlib.mk

```diff
@@ -4,9 +4,9 @@ STDLIBS_WITHIN_SYSIMG := \
 
 INDEPENDENT_STDLIBS := \
```

Comment:
@LilithHafner to answer your question. As long as the stdlib is here it is "free", but not yet upgrade-able.

Upgrade-able requires registering in the Registry and Pkg allowing it to update.

Reply:
Thanks! Is there anything that should be done on the JuliaSyntaxHighlighting side of things to enable Pkg to allow it to upgrade?